### ISSUE TITLE
Add `SetterDropCurrencyIds`

### ISSUE THREAD

**Issue by balqaasem** [setheum/setheum#179]
Check reference to #177 and #178

This could be used to determine if a currency is a currency that is eligible for `setter-cashdrops` (in Setter (SETT)) or not, for example when giving/claiming `cashdrops`.

For example (in NewRome runtime)
```
pub SetterDropCurrencyIds: Vec<CurrencyId> = vec![rSETT, ROME]; 
// add RDEX as RomeDex Dexer
```

**Comment by balqaasem** [comment]
We can just use `StableCurrencyIds` and say that -
```
let containing = T::StableCurrencyIds.contains(transacted_currency_id):
if !containing {
	Self::cashdrop_setter();
} else {
	Self::cashdrop_settcurrency(transacted_currency_id);
}

**Comment by balqaasem** [comment]
Continues on #207

**Comment by balqaasem** [comment]
Closed by #207
